backward.hpp

```diff
@@ -261,6 +261,7 @@
 #else
 #include <dlfcn.h>
 #endif
+#include <libgen.h>
```

Comment:
I expect [`libgen.h` header](https://man7.org/linux/man-pages/man0/libgen.h.0p.html) to exist when libdwarf is available, but that can be not true.